packages/cli/commands/functions/test.js

```diff
@@ -70,6 +70,11 @@ exports.builder = yargs => {
     type: 'boolean',
     default: true,
   });
+  yargs.option('log-output', {
+    describe: 'output the response body from the serverless function execution',
```

Comment:
Should we add a note that this shouldn't be used in production because it may leak PII?

Reply:
Good call, I will add this.